press/api/site_login.py

```diff
@@ -82,7 +88,6 @@ def get_product_sites_of_user(user: str):
 	sites = frappe.db.get_all(
 		"Site User", filters={"user": user, "enabled": 1}, fields=["site"], pluck="site"
 	)
```

Comment:
@Bowrna we need to get rid of this api.

First, do a initial verification of user mail, then only proceed to next ?

Or, you can do this as well
- FC doesn't need to give option to user on which site they want to login
- User goes to his own site example.frappe.cloud
- Clicked on `Login with Frappe Cloud`
- Either pass the site name while redirecting to FC or utilize http [referrer](https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Headers/Referer) header to get the site name
- Then follow the usual flow.

Only one issue is that - People can forget the site url and then become clueless. But, I don't think that's a big issue comparatively.
They can login to FC and see the site names.